creating/help.adoc

```diff
@@ -53,47 +57,62 @@ in multiple ways.
 .Sample help template in markdown format
 [source, markdown]
 ----
-% IMAGE_NAME(1)
+% IMAGE_NAME (1) Container Image Pages
 % MAINTAINER
 % DATE
 
+# NAME
+Image_name - short description
 
 # DESCRIPTION
-Describe in greater detail the role or purpose of the image.  This can include more specifics about the
-packages that make up the image.  You can also describe whether image is meant to be interactive
-or more service oriented.
+Describe how image is used (user app, service, base image, builder image, etc.), the services or features it provides, and environment it is intended to run in (stand-alone docker, atomic super-privileged, oc multi-container app, etc.).
 
 # USAGE
-Describe how to run the image as a container and what factors might influence the behaviour of the image
-itself. For example:
+Describe how to run the image as a container and what factors might influence the behavior of the image itself. Provide specific command lines that are appropriate for how the container should be run. Here is an example for a container image meant to be run by the atomic command:
 
-To set up the host system for use by the XYZ container, run:
+To pull the container and set up the host system for use by the XYZ container, run:
 
-  atomic install XYZimage
+    # atomic install XYZimage
 
 To run the XYZ container (after it is installed), run:
 
-  atomic run XYZimage
+    # atomic run XYZimage
 
 To remove the XYZ container (not the image) from your system, run:
 
-  atomic uninstall XYZimage
+    # atomic uninstall XYZimage
+
+Also, describe default configuration options (when defined): hostname, domainname, user ID run as, exposed ports, volumes, working directory, command run by default, etc.
```

Comment:
hostname, domain name are irrelevant, right? I would change "user ID run as" to "default user", same for "command run by default" -- "default command".